active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+
+    static Windows.UI.Xaml.DependencyProperty IconElementProperty{ get; };
+}
+
+unsealed runtimeclass InfoBar : Windows.UI.Xaml.Controls.ContentControl
+{
+    InfoBar();
+
+    String Title;
+    String Message;
+
+    Boolean IsOpen;
+    Boolean IsUserDismissable;
+    Boolean IsIconVisible;
+
+    Windows.UI.Xaml.Controls.Primitives.ButtonBase ActionButton;
+
+    Windows.UI.Xaml.Style CloseButtonStyle;
+    Windows.UI.Xaml.Input.ICommand CloseButtonCommand;
+    Object CloseButtonCommandParameter;
+
+    InfoBarSeverity Severity;
+    IconSource IconSource;
+
+    InfoBarTemplateSettings TemplateSettings{ get; };
+    Object Content{ get; set; };
+    Windows.UI.Xaml.DataTemplate ContentTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<InfoBar, Object> CloseButtonClick;
+    event Windows.Foundation.TypedEventHandler<InfoBar, InfoBarClosingEventArgs> Closing;
+    event Windows.Foundation.TypedEventHandler<InfoBar, InfoBarClosedEventArgs> Closed;
+
+    static Windows.UI.Xaml.DependencyProperty IsOpenProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TitleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MessageProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SeverityProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IconSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsIconVisibleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsUserDismissableProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonCommandParameterProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ActionButton{ get; };
+    static Windows.UI.Xaml.DependencyProperty ContentProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ContentTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TemplateSettingsProperty{ get; };
+}
+```
+
+## Theme Resources
+
+### Notable Theme Resources
+| Name | Description |
+|:-:|:--|
+|InfoBarHyperlinkForeground | Sets the hyperlink button text color. <br> - Note: This is set to keep hyperlinks accessible on the variously colored backgrounds defined by the severity background colors.
+|InfoBarCloseButtonStyle | Sets the close button style of the InfoBar and contains the following properties: <br> - Width <br> - Height <br> - VerticalAlignment <br> - Background <br> - Margin <br> - CornerRadius
+|InfoBarIconFontSize | Sets the icon size of the InfoBar. <br> - Note: Custom icons set via the IconSource property will not resize to 20px unless they are in the standard symbol set.
+
+### All Theme Resources
+| Resource Key | Description |
+|:-:|:--|
+| InfoBarSeverityErrorBackgroundBrush | Error Severity background color 
+| InfoBarSeverityWarningBackgroundBrush | Warning Severity background color 
```

Comment:
As the description implies, this should be ...**WarningSeverity**BackgroundBrush rather than ...SeverityWarning...